packages/api/lib/FileUtils.js

```diff
@@ -72,6 +72,7 @@ const setS3FileSize = async (s3, file) => {
 
     return { ...file, size };
   } catch (error) {
+    console.log(`File is ${file}, Error is ${error}`);
```

Comment:
Was this intentionally left in?   If so, should we use the logger package, and does it need to be JSON.stringified to be useful given how nested Core file objects are?

Reply:
When we were debugging failing integration tests for Jennifer, it was impossible to see what was really failing with logging some sort of error here. But 👍 to using the logger package

Reply:
Using logger package now - thanks for the tip